test/functionalTest/cases/3389_forced_update_option/3389_forced_update_option_batch_bool.test

```diff
@@ -0,0 +1,390 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Force update for NGSIv2 POST /v2/op/update, POST /v2/subscriptions, Dump and reset accumulator, POST /v2/subscriptions, Dump and reset accumulator, POST /v2/op/update, Dump and reset accumulator, POST /v2/op/update?options=forcedUpdate, Dump and reset accumulator.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
+# 01. POST /v2/op/update, create Slot1 and Slot2 entity with availability attribute set to true.
+# 02. POST /v2/subscriptions, to create subscription for Slot1.
+# 03. Dump and reset accumulator, see notification.
+# 04. POST /v2/subscriptions, to create subscription for Slot2.
+# 05. Dump and reset accumulator, see notification.
+# 06. POST /v2/op/update without forcedUpdate, to update Slot1 and Slot2 with the same values.
+# 07. Dump and reset accumulator, see no notification (as there was no actual update).
+# 08. POST /v2/op/update?options=forcedUpdate, to update Slot1 and Slot2 with same values.
```

Comment:
As far as I check you are covering `POST /v2/op/update?options=forcedUpdate` with actionType "update"

```
echo '08. POST /v2/op/update?options=forcedUpdate, to update Slot1 and Slot2 with same values.'
echo '========================================================================================'
payload='{
  "actionType": "update",
```

which is great :) But a checking with "append" should be also done.

It will be a matter of just repeating steps 8-9 changing updateAction to "append"

This applies to the 3 .test files 3389_forced_update_option_batch_*.test

Reply:
Fixed in 8eec791

Reply:
I have looked to 8eec791678c6f9ef22ddb7c42fe7a422c12145b2 and maybe I didn't explain it clearly...

I'm not meaning to change "update" to "append" in the current steps. What I mean is to add a new case to cover "append". That way "update" and "append" will be both covered. I mean, it should something like this:

```
# 08. POST /v2/op/update?options=forcedUpdate (actionType = update), to update Slot1 and Slot2 with same values.
# 09. Dump and reset accumulator, see notification
# 10. POST /v2/op/update?options=forcedUpdate (actionType = append), to update Slot1 and Slot2 with same values.
# 11. Dump and reset accumulator, see notification
```

(x 3 .test about POST /v2/op/update).

I hope this time have provided a better explanation :)

Reply:
fixed in 4d0f094